java/parser/src/main/java/org/sqlflow/parser/ParserGrpcServer.java

```diff
@@ -57,9 +57,31 @@ private void blockUntilShutdown() throws InterruptedException {
   static class ParserImpl extends ParserGrpc.ParserImplBase {
     @Override
     public void parse(ParserRequest request, StreamObserver<ParserResponse> responseObserver) {
-      // TODO(tony): Implement this function to call actual parsers
-      ParserResponse response = ParserResponse.newBuilder().setIndex(1).build();
-      responseObserver.onNext(response);
+      if (!(request.getDialect().equals("calcite") || request.getDialect().equals("hiveql"))) {
```

Comment:
Will the Java gRPC parser support ```"mysql"``` later?

Reply:
Nope. TiDB provided a MySQL parser in Go. :)